### ISSUE TITLE
Python 3.7 incompatibility: async is now a keyword

### ISSUE THREAD

**Issue by jonathanunderwood** [mcfletch/pyopengl#14]
In Python 3.7 `async` and `await` became keywords, so this error now occurs:

```
    from OpenGL.raw.GL.SGIX.async import *
                                ^
SyntaxError: invalid syntax
```
when doing an `import OpenGL.GL.SGIX.async`

**Comment by 5schatten** [comment]
Well any news about this issue?

**Comment by mcfletch** [comment]
Current head has changes to rename to `async_` should be in the next release.

**Comment by jayvdb** [comment]
And this fix is in the pre-release 3.1.3b2

**Comment by mcfletch** [comment]
Should be fixed with current release.
